opentelemetry-proto/README.rst

```diff
@@ -0,0 +1,19 @@
+OpenTelemetry Python Proto
+==========================
+
+|pypi|
+
+.. |pypi| image:: https://badge.fury.io/py/opentelemetry-proto.svg
+   :target: https://pypi.org/project/opentelemetry-proto/
+
+Installation
+------------
+
+::
+
```

Comment:
Would be good to have a note on how the protos were generated.

Reply:
Fixed